### DISCUSSION TITLE
Issue with faiss

### DISCUSSION THREAD

**Discussion by didopps** [FlowiseAI/Flowise#1307] [General]
Hello all
I use flowise on Render
I usually use FAISS without any isues but now I have this for all new folder add with "basepath".
before all folders created automatically.

Error: Error in faiss::FileIOReader::FileIOReader(const char*) at /app/deps/faiss/faiss/impl/io.cpp:68: Error: 'f' failed: could not open faiss.index for reading: No such file or directory

regards

**Comment by nicolasduerr** [comment]
@didopps  When using the FAISS node and provide the path to the folder with the faiss index file, I get the following error:

Error: ENOENT: no such file or directory, open '/Users/faiss/docstore.json'

**Comment by sudo-a13** [comment]
Make sure to do the initialization, not sure if this was tied to an older release. You need Upsert the vector database, that is the one-time operation to prepare your data, one that is completed, you can connect and run it independently.

In the Flowise 2.2.3 UI there is Green Icon next to the chat button to do the initialization.

**Reply by coltonbatts** [reply]
This is the way.
